checks/python_module_version.go

```diff
@@ -0,0 +1,68 @@
+package checks
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+func init() {
+	availableChecks["python-module-version"] = func(args map[string]interface{}) (Checker, error) {
+		return PythonModuleVersion{}.FromArgs(args)
+	}
+}
+
+// PythonModuleVersion checks if python module is installed on the system
+// And verifies its version.
+//
+// Type:
+//   - python-module-version
+//
+// Support Platforms:
+//   - Linux
+//   - Windows
+//
+// Argument:
+//   name (required): A string value that represents the gem name
+//   version (optional): An optional version number to check
```

Comment:
@chasinglogic pointed out to me the standard is leaving " (optional)" out to indicate optional